ast/src/parsed/asm.rs

```diff
@@ -203,6 +224,13 @@ impl AbsoluteSymbolPath {
         parts.push(part.to_string());
         Self { parts }
     }
+
+    /// Formats the path without leading `::` and uses `.` as separator if
+    /// there are at most two components.
+    pub fn to_dotted_string(&self) -> String {
```

Comment:
This looks the same as the other `to_dotted_string` impl above?

Reply:
It is, but this is for AbsoluteSymbolPath and the other is for SymbolPath.